tests/test_all.py

```diff
@@ -273,9 +273,16 @@ def test_no_downgrade_on_install(container: ContainerData) -> None:
     of any package already installed in the container as that would throw
     a question to the user and break the builds.
     """
-
     conn = container.connection
-    conn.run_expect([0], "timeout 2m zypper ref && zypper -n in libsolv-tools")
+
+    zypper_ref = conn.check_output("timeout 2m zypper ref")
+    # Temporary check for bsc#1243360
+    assert (
+        "Skipping service 'container-suseconnect-zypp' because of the above error"
```

Comment:
well, `test_no_downgrade_on_install`is supposed to run without authentication for container-suseconnect ..

Reply:
Check the verification runs, the 15-SP6 catches the container-suseconnect error, the SLES 16 test run is green. This should show that it works as intended or am I missing something?